content/docs/rfcs/template/README.md

```diff
@@ -5,18 +5,19 @@ name: (Waku v2 | Logos | Codex) RFC Template
 status: (raw|draft|stable)
 category: (Standards Track|Informational|Best Current Practice)
 tags: an optional list of tags, not standard
+description: an optional short summary
```

Comment:
I'd either set it to a meaningful value, or to the abstract as suggested in https://discord.com/channels/864066763682218004/1179553924617871422/1180115380765728829, or don't set it at all.

Reply:
as far as the actual limit goes, I don't think it has been reviewed yet https://discord.com/channels/@me/1179730970946043935/1181715830094970900

Reply:
@felicio Moving conversation to https://github.com/vacp2p/rfc/pull/645